### PR TITLE
Document preparing external locations when creating catalogs

### PR DESCRIPTION
**Pull request by JCZuurmond**
## Changes
Document preparing external locations when creating catalogs.

### Functionality

- [x] update `README.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -563,10 +563,18 @@ Once the upgrade is completed, these principals can (and should) be deleted.
...
 ##### Step 2.5: Create Catalogs and Schemas
-In this step we will create the UC catalogs and schemas required for the target tables.
-The `create-catalogs-schemas` [UCX command](#create-catalogs-schemas-command) can be used to create the UC catalogs and schemas.
-
-The command will create the UC catalogs and schemas based on the mapping file created in the previous step.
+In this step, we will create the UC catalogs and schemas required for the target tables using the
+[`create-catalogs-schemas` command](#create-catalogs-schemas-command). The command will create the UC catalogs and
+schemas based on the mapping file created in the previous step.
+
+This step requires considering how to [physically separate data in storage](https://docs.databricks.com/en/data-governance/unity-catalog/best-practices.html#data-is-physically-separated-in-storage)
```
[the remark this answers is not in the document]
Comment (biswadeepupadhyay-db) [reply README.md:570]: Ok so as long as we are not going ahead with the default metastore path (used at the time of UC setup) it is recommended that we create external storage and storage credential for the metastore paths we aim to use for our catalogs.

[the remark this answers is not in the document]
Comment (JCZuurmond) [reply README.md:570]: The later is generally advised by Databricks (according to the linked docs).

**Review by nfx — APPROVED** [review]
lgtm

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (biswadeepupadhyay-db) [reply README.md:570]: Also please confirm if step 2.1 till step 2.3 are related to finding the external locations from the assessment related to hive external tables and creating corresponding external location and storage credentials related to those external paths.

**Pull request merged**